test/e2e_node/log_path_test.go

```diff
@@ -66,8 +65,8 @@ var _ = framework.KubeDescribe("ContainerLogPath", func() {
 				framework.ExpectNoError(err, "Failed waiting for pod: %s to enter success state", logPodName)
 
 				// get containerID from created Pod
-				createdLogPod, err := podClient.Get(logPodName)
-				logConID := kubecontainer.ParseContainerID(createdLogPod.Status.ContainerStatuses[0].ContainerID)
+				pod, err := podClient.Get(logPodName)
```

Comment:
I think the pod returned by `client.Create` does not have enough `status` value we needed to parse container ID. So I `Get` it after it becomes running. Am I right?